tests/conftest.py

```diff
@@ -909,18 +911,29 @@ def tier_marks_name():
 
 @pytest.fixture(scope='function', autouse=True)
 def health_checker(request, tier_marks_name):
+    def finalizer():
+        try:
+            ceph_health_check_base()
+            log.info("Ceph health check passed at teardown")
+        except CephHealthException:
+            log.info("Ceph health check failed at teardown")
+            # Retrying to increase the chance the cluster health will be OK
+            # for next test
+            ceph_health_check()
+            raise
```

Comment:
Just thinking if we can suppress 'raise' here in case above check passed?

Reply:
The idea is to fail the test if we are in the 'except' block